### PR TITLE
Return Buffer instead of Array

### PR DESCRIPTION
**Pull request by dcousens**

### CODE REVIEW

**Comment by jprichardson** [comment]
> I think the risk of this being passed interchange-ably to hash functions and other encoding functions which expect a Buffer, is much worse.

I think this point by @dcousens is worth mentioning. I agree on this basis.

File: index.js [hunk 1]
```diff
@@ -68,12 +68,12 @@ module.exports = function base (ALPHABET) {
       bytes.push(0)
     }
 
-    return bytes.reverse()
+    return new Buffer(bytes.reverse())
```
Comment (fanatid) [inline index.js]: `Buffer.from`?

**Comment by dcousens** [comment]
Changed to `Buffer.from`

**Comment by fanatid** [comment]
Tests in #29

**Comment by dcousens** [comment]
:+1:  awesome @fanatid

**Comment by dcousens** [comment]
Release? :+1:

**Comment by jprichardson** [comment]
> Release? 👍

Yep, I'd do it... but I'm not an owner on npm ;) Will you add me?

**Comment by dcousens** [comment]
@jprichardson ha,  oops.   I thought you were.

Published as `v2.0.0` and added you as an owner

**Pull request merged**